### DISCUSSION TITLE
Design Refresh / Styleguide

### DISCUSSION THREAD

**Discussion by thescientist13** [ProjectEvergreen/www.greenwoodjs.dev#6] [Ideas]
## Overview

Just starting a discussion to solicit thoughts and feedback on the design, theme, and overall aesthetic for the new website.  Currently meeting with a designer now so will update here as revisions come in, but please free to discuss here, in particular feedback on the current site.  👍 

## Styleguide

Will want to define things like:
1. [ ] Color Palette (something that can build off the leaf)
1. [ ] What font should we use?  Could try the new Vercel font [Geist](https://vercel.com/design/introduction) or [Inter](https://fonts.google.com/specimen/Inter)?
1. [ ] Components
    - Code snippet w/ copy to clip board
    - Table
    - Quotes / Info Banners
    - Links
    - External Links

## Dark Mode
Probably good for a website, especially documentation website, to support Dark Mode.  I don't know too much but Syntax.fm did a great [podcast](https://podcasts.apple.com/us/podcast/syntax-tasty-web-development-treats/id1253186678?i=1000634136355) on the topic.

## Ecosystem
Having this will be helpful so we can style all adjacent projects like the getting started rep and starter kits.
- https://github.com/ProjectEvergreen/wcc
- https://github.com/ProjectEvergreen/greenwood-getting-started
- https://github.com/ProjectEvergreen/greenwood-demo-adapter-netlify
- https://github.com/ProjectEvergreen/greenwood-demo-adapter-vercel
- https://github.com/ProjectEvergreen/greenwood-template-blog
- https://github.com/thescientist13/greenwood-demo-adapter-vercel-lit
- https://github.com/thescientist13/greenwood-htmx
- https://github.com/ProjectEvergreen/greenwood/tree/master/packages/init

**Comment by thescientist13** [comment]
Got a decent example I think I'm liking, imagine it with the syntax highlighting from https://github.com/ProjectEvergreen/www.greenwoodjs.dev/pull/16 and the slideshow from #17 
![Screenshot 2024-02-12 at 9 45 01 PM](https://github.com/ProjectEvergreen/www.greenwoodjs.dev/assets/895923/2c3ed3c5-f71c-40fa-afaa-c581635f1df6)
![Screenshot 2024-02-12 at 9 39 25 PM](https://github.com/ProjectEvergreen/www.greenwoodjs.dev/assets/895923/687b6e6b-2469-4a0f-87a9-1e59c252b828)
![Screenshot 2024-02-12 at 9 39 34 PM](https://github.com/ProjectEvergreen/www.greenwoodjs.dev/assets/895923/e81a2cd3-a899-4245-aa44-72fc7cd1968c)

**Comment by thescientist13** [comment]
Here's what I'm leaning to the most
https://deploy-preview-18--super-tapioca-5987ce.netlify.app/

Some odds and ends to clean up with animations, spacing, gradients and of course overall vibe, but I think it's an aesthetic I am happy with.

**Comment by Auhseh** [comment]
Currently working on the Style Guide, Here's the initial exploration;

https://www.figma.com/file/ePhyD5qNMrBAcHwQ08BnQC/GreenWood?type=design&node-id=1%3A2&mode=design&t=400dqKMaa6XxjPvW-1

**Reply by thescientist13** [reply]
Thanks for this!  I think other than seeing if we can explore moving away from the black (to maybe a deep blue?) this is looking really good.  And if we can try one version with the Geist font.

**Reply by Auhseh** [reply]
Great Suggestion!
I have added the Geist font and I’ve added the shade of blue you used in the test deploy.

I perfectly understand your thoughts around the Vercel resemblance, but my humble opinion is that, in trying to build a cohesive brand & style guide around Greenwood, the Blue kinda messes up the harmony almost like there are 2 primary 
(We can use it for WCC?)

Vercel black is #000000, to address your concerns, I changed the black to #0E0F0C which has a green hue, also I changed the background from white to #EDEFEC which also has a green hue to take it farther from vercel.

Here are both of them side by side.
https://www.figma.com/file/ePhyD5qNMrBAcHwQ08BnQC/GreenWood?type=design&node-id=113%3A740&mode=design&t=Oxojzr9GhCspVw0b-1

**Reply by Auhseh** [reply]
Overall I have made considerable progress on the page structure, We should start exploring ideas for the copy.

**Reply by thescientist13** [reply]
Thank you so much, and so let me digest the side-by-side.  I do like it a bit more than the black.  With project being Greenwood and that connotation of evergreen browsers, a blue feels a bit more thematically aligned than black.  (but I guess all life springs from carbon, which is traditionally a black color 😅 )

One thing that might help bring the blue into place is the syntax-highlighting, in which in the PR I was using the Nord theme, which might be worth incorporating
- https://github.com/ProjectEvergreen/www.greenwoodjs.dev/pull/18
- https://marketplace.visualstudio.com/items?itemName=arcticicestudio.nord-visual-studio-code

Here the various Prism themes for consideration
- https://prismjs.com/
- https://github.com/PrismJS/prism-themes

I think it might still be good to keep the headings text as black (like for the hero heading).  I guess ultimately we should consider whatever one would likely play the best with contrast accessibility and dark mode.

**Reply by Auhseh** [reply]
I get your preference for the blue 😅
I'll try to make it work

**Reply by thescientist13** [reply]
Had a chance to get some feedback on the team and they don't feel the black is too Vercel-y hah, and agree that too many accent colors could start to complicate things.  Perhaps not so much of a pitch black though, maybe a charcoal or similar direction might look nice, but otherwise yeah I think let's work on making the black + light green work for that section instead of the blue.  👍 

Appreciate you looking into it.  I'll also start playing around with syntax highlighting colors just to see what good options might align with that theme.

**Reply by Auhseh** [reply]
Bigups to the team 🙌😅

**Comment by thescientist13** [comment]
Ok, between #23 and #25 and the work done in #4 and #5  , I think we can branch out to those issues for future tracking and close this one out.
